src/JavascriptVM/vm.spec.ts

```diff
@@ -11,7 +11,7 @@ describe("javascript vm", () => {
   });
 
   it("calls the highlightBlock callback", () => {
-    const code = "highlightBlock('abc');";
+    const code = "function start() {highlightBlock('abc');}";
```

Comment:
Similar to the required updates to the tests, I think it would be best to update the example programs, so that they can be loaded and run w/o modification? If you prefer that we do this via a follow-up PR then that's fine too.

Reply:
done!